docs/tutorial/tut_bridges.md

```diff
@@ -144,9 +148,54 @@ Below there are described the different bridge aliases, links table and column n
 10. The name of the column inside the `EFF_SAT_CUSTOMER_ORDER` table describing the timestamp when a `CUSTOMER_ORDER` relationship ended is `END_DATE`.  
 11. The name of the column inside the `EFF_SAT_CUSTOMER_ORDER` table recording the load date/timestamp of a `CUSTOMER_ORDER` relationship is `LOAD_DATE`.
 
+In a similar fashion, continue defining the different aliases for the `ORDER_PRODUCT` link and effectivity satellite columns. 
+
 The dbt_project.yml below only defines two link relationships but to add others you would follow the same method inside 
 the bridge_walk metadata. For instance, it can be seen where the `PRODUCT_COMPONENT` relationship metadata would begin.
 
+=== "dbt_project.yml"
```

Comment:
We no longer use vars in a `dbt_project.yml` Please remove this.

Reply:
Resolved